### PR TITLE
1183-UI-Development-Saff-Search

### PR DESCRIPTION
**Pull request by stevenc987**
Signed-off-by: IDIR\schen <steven.chen@gov.bc.ca>

*Issue #:*
https://github.com/bcgov/entity/issues/1183

*Description of changes:*

*UI Code Review Checklist:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/store/modules/business/index.ts [hunk 1]
```diff
@@ -76,6 +78,16 @@ export default class BusinessModule extends VuexModule {
     this.context.dispatch('getOrganizations', null, { root: true })
   }
 
+  @Action({ rawError: true })
+  public async staffSearch (businessNumber: string) {
```
Comment (pksumesh) [inline auth-web/src/store/modules/business/index.ts]: Change the function name to something like searchBusiness.

File: auth-web/src/components/auth/StaffSearchForm.vue [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  businessStore = getModule(BusinessModule, this.$store)
+
+  businessNumber: string = ''
+  errorMessage:string = ''
+
+  private isFormValid (): boolean {
+    return this.$refs.form.validate()
+  }
+
+  async staffSearch () {
```
Comment (pksumesh) [inline auth-web/src/components/auth/StaffSearchForm.vue]: Change the function name to something like searchBusiness.

File: auth-web/src/components/auth/SearchBusinessForm.vue [hunk 3]
```diff
@@ -0,0 +1,111 @@
+<template>
+    <v-container>
+      <div class="view-container">
+        <article>
+          <h1>Search Co-operatives</h1>
+          <p class="intro-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu condimentum sed.</p>
```
Comment (leksmall) [inline auth-web/src/components/auth/SearchBusinessForm.vue]: Is the text here yet to be finalized?

Comment (stevenc987) [reply auth-web/src/components/auth/SearchBusinessForm.vue]: Yes, I did unit test after I changed function name.

The text you mentioned is temporary. Should I remove it and just keep it empty there?

Comment (stevenc987) [reply auth-web/src/components/auth/SearchBusinessForm.vue]: Sorry, I was confused what you were talking about unit test. I will add unit test for StaffSearchForm component soon. Thanks.

Comment (leksmall) [reply auth-web/src/components/auth/SearchBusinessForm.vue]: > Yes, I did unit test after I changed function name.
> 
> The text you mentioned is temporary. Should I remove it and just keep it empty there?

Either way is fine.

**Comment by stevenc987** [comment]
OK. Unit test for searchbusiness component is there now. Please take  a look.

**Pull request merged**